### ISSUE TITLE
Stanford Social Innovation Article

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#707]
https://ssir.org/
https://ssir.org/articles/entry/artificial_intelligence_as_a_force_for_good#

**Comment by danelsuga** [comment]
Reading up to start on this.

**Comment by danelsuga** [comment]
I had a meeting with Gideon the other day to hear his experience and thoughts. I'm reading now and plan to forge an outline shortly. Maybe we should brainstorm that together, or should I just give it a go myself?

**Comment by laurenluz** [comment]
I think it would be awesome if you got started with something and then we had a meeting to go over it/add to it? wdyt?

**Comment by danelsuga** [comment]
Moving along slowly but surely. Hope to have something for your review @laurenluz in a week or so (maybe longer on account of EthBCN).

**Comment by danelsuga** [comment]
On hold due to ETHBCN and ETHCC.

**Comment by danelsuga** [comment]
Summer events and family tragedy put this on hold, but I want to get back to it now.

**Comment by danelsuga** [comment]
Actively working on this.

**Comment by laurenluz** [comment]
Hey @danelsuga I'd love to have a call about this an support you. See where you're at, throw some ideas around. I'll be back to calls hopefullly next week and we can chat them. In the meantime, could you post a google doc with some of your ideas? Maybe start up an outline?

**Comment by danelsuga** [comment]
Will do!

**Comment by danelsuga** [comment]
I am advancing on this article.

**Comment by danelsuga** [comment]
On hold for a couple more urgent things (project owners course and what is giveth? roadmap updates).

**Comment by danelsuga** [comment]
This piece is getting some attention again. :D

**Comment by danelsuga** [comment]
outline: https://docs.google.com/document/d/1ztsiz5pgVg_yEbvX9DEAuf6gi1DD6NKbvlkZKht1F6k/edit?usp=sharing

**Comment by laurenluz** [comment]
Hey @danelsuga!

love the thesis & I think a lot of the right ideas are there. I wasn't able to fully understand the outline because I don't know which quotes you are talking about.

Anyway, I left some comments thoughts, and at the bottom I put a kind of suggested outline. One thing, from 1st glace at your outline... it looks like you go back and forth a few times around regen economies and make it could be reordered a little to clearly outline the problem & the possible solution... starting out with an intro, then examples, the projections into the future.

Lmk if that makes sense.

**Comment by danelsuga** [comment]
Thanks! 

As for not knowing the quotes, etc., it's possible just to look at the article draft if you want, but I can also flesh out the outline too if you prefer.

Yes, I do this "back and forth" on purpose for the sake of Giveth. I cut to the chase of providing two examples at the start to demonstrate how blockchain regen econ can change everything. Then I have to go back to the basics and describe the state of the current structure ("problem") and then provide the blockchain explanation ("solution). I start with the examples of Giveth and Commons Stack in order to have that out there straight away and to be able to come back to it several times throughout.

Honestly, I don't think your outline looks much different from mine. You've just removed detailed elements for my paragraphs. It may be the case that mine seems back and forth-y because it's only an outline that you're looking at. In the article itself, you will see how it plays out. I want to get the examples in there asap, and that seems to work with what you say in your outline about the "blockchain introduction". You've just eliminated the examples, but this would still need to be an explanatory introduction paragraph. In any case, I think the examples are totally essential (and the point) for our purpose with this piece, but they can be removed if you feel strongly about it.

**Comment by laurenluz** [comment]
the link to the draft article is not in this issue or in the doc you shared.
I was assuming that you were making the outline to zoom out a little and reorganize the content.

I don't think you should remove the examples.

**Comment by danelsuga** [comment]
Ugh sorry!! Will do shortly.

**Comment by danelsuga** [comment]
Hi @laurenluz !

Wow, I did some cutting. And it's still too long. Needs to be 800-1000, and it's currently about 1200.

https://docs.google.com/document/d/1y5Pek2vVavNL_J56sv-ZzgEbK6_vg_KnArojU0IDL8c/edit

**Comment by danelsuga** [comment]
(The quotes all need to be credited differently and streamlined, I know.)

**Comment by danelsuga** [comment]
Gideon and Griff have shared edits, which are being incorporated.

**Comment by danelsuga** [comment]
Waiting for new feedback from @GriffGreen

**Comment by brichis** [comment]
@danelsuga Griff already review the document :)

**Comment by danelsuga** [comment]
Yes thanks @brichis I know. I should have updated the comments.

**Comment by danelsuga** [comment]
submitted.

**Comment by laurenluz** [comment]
YES GIRL!!!!!!!!!!!

**Comment by laurenluz** [comment]
What happened here? did they accept it? I would close the issue only after it is published/rejected... if it's rejected we should publish elsewhere

**Comment by danelsuga** [comment]
That's correct. It's submitted. Didn't think it's a big deal to move the issue out of the way. Obviously I would update it when information comes it. No biggie.

**Comment by laurenluz** [comment]
it's not in the way if it's not a fully completed and there is more work/need for updates

**Comment by laurenluz** [comment]
@danelsuga - what's the status on this? when will we hear back from them?

Also! if they don't accept it, we should look into what other publications might be interested in this and submit it. I would love to see this go up somewhere.

**Comment by danelsuga** [comment]
It was rejected without any feedback.

**Comment by danelsuga** [comment]
Request for feedback remains ignored.
Checking in for other possible placement opps.

**Comment by danelsuga** [comment]
Generic feedback arrived: 

SSIR has indicated the piece was not critical. They cited their article [Web3 and the Trap of For-Good](https://ssir.org/articles/entry/web3_and_the_trap_of_for_good) as a strong example to consider. They want "some explanation of the subject and examples, but also stepping back to think about the big picture."

I'm going to close this issue now and open a new issue for the new solicited journal, which will be [The Chronicle of Philanthropy](https://www.philanthropy.com/).
